### ISSUE TITLE
Metabase 0.47.5 Pivot Table "There was a problem displaying this chart"

### ISSUE THREAD

**Issue by Spinwing** [metabase/metabase#35018]
### Describe the bug

A simple pivot table question on a dashboard stopped working showing "There was a problem displaying this chart" just after upgrading to 0.47.5. Switching back to 0.47.4 fixed the issue. 

I will try to produce some sample data to reproduce the issue ASAP, in the meantime I wanted to raise an early alert of the regression issue. 

### To Reproduce

- Upgrade to 0.47.5
- Open a dashboard with a Pivot Table

### Expected behavior

Expecting Pivot Table to work as usual

### Information about your Metabase installation

```JSON
- Browser: Safari 17, Brave 1.59.120
- Client: OSX 14.0
- Server: Rocky Linux 8.8
- Database: MariaDB 10.3.39-MariaDB
- Metabase: 0.47.5
- Hosting: jar file
- Internal DB: MariaDb  10.3.39-MariaDB
```

### Severity

Blocking an upgrade

**Comment by uladzimirdev** [comment]
@Spinwing thanks! FYI we have different tests for main scenarios and they cover Pivot table, so without clear steps to reproduce the issue we can’t help.

**Comment by noahmoss** [comment]
I merged https://github.com/metabase/metabase/pull/34583 in 47.5 so it's possible there is a regression caused by that. However I just tried creating a pivot table using the sample dataset and adding it to a dashboard and it seems to work fine. If you have a way to repro, and/or you can provide any relevant backend logs, I will look into it.

**Comment by zbodi74** [comment]
A similar issue with reproduction is here: https://github.com/metabase/metabase/issues/35025

**Comment by noahmoss** [comment]
Closing in favor of https://github.com/metabase/metabase/issues/35025

**Comment by Spinwing** [comment]
As to leave trace of my analysis, also in light of #35035 

- simple question was based on uploaded CSV. Metabase then created table onto the MariaDB database
- we are not using H2, everything was moved to MariaDB running locally
- there was no nested question, just a simple count of records grouped by two text fields
- I tried recreating the question from scratch, same error. This does not happen on 47.4

I will try to anonymize data and post it asap.
